os_migrate/plugins/module_utils/security_group.py

```diff
@@ -26,6 +26,7 @@ class SecurityGroup(resource.Resource):
 
     @classmethod
     def from_sdk(cls, conn, sdk_resource):
+        logger.get_logger("from_sdk").debug("Fetching data from SDK")
```

Comment:
We probably need to discuss this more, but i don't think our primary focus should be on this kind of logging? IMO primarily we should be saving the Ansible run logs, so probably using something like: https://docs.ansible.com/ansible/latest/reference_appendices/config.html#default-log-path ?

BTW this is M4 task in Jira board. We can discuss if we think the priorities should change on anything, but i'm not convinced we should be focusing on this until all M3s are either done or have assignees working on them?